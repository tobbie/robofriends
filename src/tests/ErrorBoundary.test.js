import { shallow } from "enzyme";
import React from "react";
import ErrorBoundary from "../components/ErrorBoundary";

describe("error boundary tests", () => {
  it("expect to render error boundary component", () => {
    expect(shallow(<ErrorBoundary />)).toMatchSnapshot();
  });

  it("expect hasError to be false", () => {
    expect(shallow(<ErrorBoundary />).state()).toEqual({ hasError: false });
  });
});
