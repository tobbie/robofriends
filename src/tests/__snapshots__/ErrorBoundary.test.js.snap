// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error boundary tests expect to render error boundary component 1`] = `ShallowWrapper {}`;
